src/main/java/com/microsoft/aad/msal4j/IBroker.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.aad.msal4j;
+
+/**
+ * Used to define the basic set of methods that all Brokers must implement
+ */
+public interface IBroker {
```

Comment:
The I prefix is quite an outdated approach. I would suggest you consider calling this `Broker` instead.

Reply:
I would argue to not follow conventions of another language, if you can help it.

Reply:
I also agree that it's outdated, but it's also something that all the other interfaces in our library follow. Consistency with the other MSALs is also a big focus for this feature, since most of the libraries will be also be integrating with MSALRuntime we want to keep things as similar as we can